CONTRIBUTING.es.md

```diff
@@ -0,0 +1,176 @@
+Como comunidad le damos la bienvenida a cualquiera que desee escribir y enviar entradas al blog de Playful Programming. ¡Vamos! En este documento revisaremos los pasos necesarios para crear una nueva publicación, y después enviarla a PP como una *pull request* o solicitud para incorporar cambios.
+
+> [!NOTA]
+> Para un tutorial más general sobre cómo contribuir a un proyecto con GitHub podrías revisar la guía [ Primeras contribuciones](https://github.com/firstcontributions/first-contributions/blob/main/translations/README.mx.md) antes de seguir leyendo esta página.
+
+Algunos puntos a tomar en cuenta cuando escribas tu artículo:
+- ¡Sé incluyente! En PP apoyamos a los desarrolladores de todos los niveles de experiencia - no excluyamos a los novatos ni desanimemos a los lectores que desean aprender algo nuevo.
+- Busca proporcionar información relevante y verdadera - ¡Te recomendamos citar las fuentes de tu trabajo!
+- Mantén tu contenido imparcial; es decir, no hagas anuncios de productos o servicios sin ninguna razón.
+
+Si en algún momento te sientes bloqueado o tienes alguna duda que quieras resolver, no dudes en [abrir un reporte de problema en GitHub](https://github.com/playfulprogramming/playfulprogramming/issues/new) o bien [comunícate con nosotros en Discord](https://discord.gg/FMcvc6T). ¡Estaremos encantados de ayudarte!
+
+---
+
+Contenido:
+1. [Crear un perfil de autor](#crear-un-perfil-de-autor)
+2. [Escribir una nueva entrada](#escribir-una-nueva-entrada)
```

Comment:
I think "publicación" is a better translation for "post". I'd suggest updating all occurrences

```suggestion
2. [Escribir una nueva publicación](#escribir-una-nueva-publicación)
```